src/Microsoft.AspNet.SignalR.SqlServer/SqlServerServicesBuilderExtensions.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNet.SignalR;
+using Microsoft.AspNet.SignalR.Messaging;
+using Microsoft.AspNet.SignalR.SqlServer;
+using Microsoft.Framework.ConfigurationModel;
+
+namespace Microsoft.Framework.DependencyInjection
+{
+    public static class SignalRServiceCollectionExtensions
+    {
+        public static SignalRServicesBuilder AddSqlServer(this SignalRServicesBuilder builder, Action<SqlScaleoutConfiguration> configureOptions = null)
+        {
+            return builder.AddSqlServer(configuration: null, configureOptions: configureOptions);
+        }
+        public static SignalRServicesBuilder AddSqlServer(this SignalRServicesBuilder builder, IConfiguration configuration, Action<SqlScaleoutConfiguration> configureOptions = null)
+        {
+            builder.ServiceCollection.AddSingleton<IMessageBus, SqlMessageBus>();
```

Comment:
Before this method was using a `ServiceDescriber`. I think we should continue using that.